package.json

```
@@ -25,6 +25,7 @@
     "cli-table3": "0.6.0",
     "compare-versions": "3.6.0",
     "escape-string-regexp": "4.0.0",
+    "ignore": "^5.1.8",
```

Please set it fixed and remember to update the `yarn.lock`

```suggestion
    "ignore": "5.1.8",
```

---

done